python/ray/_private/async_compat.py

```diff
@@ -19,6 +19,14 @@ def get_new_event_loop():
         return asyncio.new_event_loop()
 
 
+def try_install_uvloop():
+    """Installs uvloop as event-loop implementation for asyncio (if available)"""
+    if uvloop:
+        uvloop.install()
+    else:
+        pass
```

Comment:
What does this do exactly?

Ray will already use the `uvloop` implementation if it's available in`get_new_event_loop()` above -- do we need both?

Reply:
This replaces default asyncio policy so that asyncio internally instantiates uvloop as well (whenever you call `get_event_loop` for ex)

Reply:
I see but isn't this redundant with the above?

Reply:
Yeah, so these things allow us to control the event-loops that we create, but installing the t/h asyncio policy allows us to guarantee that *any* event-loop created using standard asyncio API would be uvloop.